app/models.py

```diff
@@ -3025,6 +3047,80 @@ class BriefAssessor(db.Model):
     email_address = db.Column(db.String)
 
 
+class Team(db.Model):
+    __tablename__ = 'team'
+
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String, nullable=False)
+    email_address = db.Column(db.String)
+    status = db.Column(
+        db.Enum(
+            *[
+                'created',
+                'completed',
+                'deleted'
+            ],
+            name='team_status_enum'
+        ),
+        nullable=False
+    )
+
+    created_at = db.Column(DateTime, index=False, nullable=False, default=utcnow)
+    updated_at = db.Column(DateTime, index=False, nullable=False, default=utcnow, onupdate=utcnow)
+
+    team_members = relationship('TeamMember', cascade="all, delete-orphan")
+
+
+class TeamBrief(db.Model):
+    __tablename__ = 'team_brief'
+
+    id = db.Column(db.Integer, primary_key=True)
+    brief_id = db.Column(db.Integer, db.ForeignKey('brief.id'), nullable=False)
+    team_id = db.Column(db.Integer, db.ForeignKey('team.id'), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+
+    user = relationship('User')
+    team = relationship('Team')
+
+
+class TeamMember(db.Model):
+    __tablename__ = 'team_member'
+
+    id = db.Column(db.Integer, primary_key=True)
+    is_team_lead = db.Column(db.Boolean, default=False, nullable=False)
+    team_id = db.Column(db.Integer, db.ForeignKey('team.id'), nullable=False)
```

Comment:
It turns out a foreign key in SQLAlchemy does not automatically add an index, so this and other fkey columns may need `index=True`.